py/picca/delta_extraction/data_catalogues/desi_data.py

```diff
@@ -180,13 +180,13 @@ def set_blinding(self, is_mock):
         is_mock: boolean
         True if reading mocks, False otherwise
         """
-        # blinding checks
+        # do not blind mocks
         if is_mock:
-            if self.blinding != "none":  # pragma: no branch
-                self.logger.warning(f"Selected blinding, {self.blinding} is "
-                                    "being ignored as mocks should not be "
-                                    "blinded. 'none' blinding engaged")
-                self.blinding = "none"
+            self.blinding = "none"
+        # do not blind metal forests (not lya)
+        elif Forest.log_lambda_rest_frame_grid[0] < ABSORBER_IGM["LYA"]:
+            self.blinding = "none"
```

Comment:
Wait, I think this should be the opposite! We should blind rest-frame grids < 1215.67

Reply:
ups, you're right. I have fixed it